monster.py

```diff
@@ -40,21 +40,24 @@ def think(self):
 
 
 class Hunter(Monster):
-    def __init__(self, room, x=0, y=0):
+    def __init__(self, room, x=0, y=0, speed=0.15):
```

Comment:
Preco je to parameter konstruktora, ked sa to aj nikdy nebude nastavovat?
(objekty vyrabame nacitanim z mapy, pricom sa kazdemu podhodia 3 veci - room, x, y)
Asi zatial staci, ked to bude v tele funkcie ... niekedy sa to moze zrefaktorovat, ze by sme mali nejaky config s konstantami vsetkych priser ... ale zatial ich mame len zopar, tak konstanty v kode az tak nevadia ...